tests/c2pa.py

```
@@ -1531,17 +1545,19 @@ def sign(self, signer: Signer, format: str, source: Any, dest: Any = None) -> Op
                 signer._signer,
                 ctypes.byref(manifest_bytes_ptr)
             )
-            
+
             if result < 0:
                 _handle_string_result(_lib.c2pa_error())
```

Here, we can silently fail